### ISSUE TITLE
Entity won't be removed in ForwardExtentCopy

### ISSUE THREAD

**Issue by Erumeldor** [boy0001/FastAsyncWorldedit#1034]
# Bug report

**Debug paste link**:
https://hastebin.com/etudewabom.rb
Some (unrelevant) plugins removed for privacy reasons.

**Description of the problem:**
So, I'm pasting a part of a backup world into the original world.
Therefore, I set up two EditSessions, and perform a ForwardExtentCopy. I need to perform it manually, because I want to edit the copied blocks. This is done in method 'replaceBlocks()', source linked below.
My problem is, that every entity of the world where I paste won't be removed. So, here are some pictures, before and after the action.
Before: https://image.prntscr.com/image/ofPIuJCQSnir5-b_aZOtqw.png
After: https://image.prntscr.com/image/L5OWqJTBQLOg9ug5e1lZYg.png
Before: https://image.prntscr.com/image/JsEjpfmeTOSYmtqHjVw67g.png
After: https://image.prntscr.com/image/umqcrGjZQXC5VA5rJYYicw.png

I happens with kind of every entity: Animals, villager, minecarts, itemframes, armorstand, ...
Here is my source-code. 
https://hastebin.com/oxaliporiy.cs

'BlockCopy' is a class and will be instanced, 'action' will be performed afterwards.
Is there a mistake in my code, or is it a bug with FAWE?
I tried many different ways of doing this, and the way I'm doing it now seems to be the best.
Thanks for your help.

**How to replicate**:
See my sourcecode.

**Checklist**:

- [X] I included a `/fawe debugpaste` link
- [X] I made sure there aren't duplicates of this report [(Use Search)](https://github.com/boy0001/FastAsyncWorldedit/issues?utf8=%E2%9C%93&q=is%3Aissue)
- [X] I made sure I am using an up-to-date version of FAWE
- [X] I Made sure the bug/error is not caused by any other plugin (I mean, it is my plugin that causes the error, but I'm not sure if thats because of FAWE or me.)

**Comment by N0tMyFaultOG** [comment]
If the issues persists still, reopen the ticket.
